dashboard/src/app/components/connexion/connexion.component.ts

```diff
@@ -12,9 +12,13 @@ export class ConnexionComponent implements OnInit {
   public email!: string;
   public password!: string;
   public souvenir: boolean = false;
-  constructor(public router: Router, public authServe: AuthService) {}
+  public numberOfOrders = 0;
```

Comment:
I would rather go for naming it : blabla

Reply:
i don't like blabla as name